docs/SUMMARY.md

```diff
@@ -13,11 +13,11 @@
 
 ## Examples
 
-- [Overview](examples/README.md)
-- [Simple connect](examples/01_simple_connect/README.md)
-- [Listen to blocks](examples/02_listen_to_blocks/README.md)
-- [Listen to balance change](examples/03_listen_to_balance_change/README.md)
-- [Generate Accounts](examples/04_generate_account/README.md)
-- [Read Storage](examples/05_read_storage/README.md)
-- [Craft Extrinsic](examples/06_craft_extrinsic/README.md)
-- [Transfer DOTs](examples/07_transfer_dots/README.md)
+- [RPC](examples-rpc/README.md)
```

Comment:
We need the same examples for ApiRx & ApiPromise (The straight RPC should be deprecated/removed from the index)